src/main/webapp/app/entities/result/result-detail.component.ts

```diff
@@ -29,9 +29,9 @@ export class ResultDetailComponent implements OnInit {
         this.resultService.getFeedbackDetailsForResult(this.result.id).subscribe(res => {
             this.result.feedbacks = res.body!;
             this.feedbackList = res.body!;
-            if (!this.feedbackList || this.feedbackList.length === 0) {
-                // If we don't have received any feedback, we fetch the buid log outputs
-                this.repositoryService.buildlogs(this.result.participation!.id).subscribe((repoResult: BuildLogEntry[]) => {
+            if ((!this.feedbackList || this.feedbackList.length === 0) && this.result.participation) {
```

Comment:
I check whether `this.result.participation` is available to fix this issue:
https://sentry.io/share/issue/b33955fb9118483b8c9abe62407df745/